flathub.json

```diff
@@ -1,3 +0,0 @@
-{
-  "only-arches": ["aarch64", "x86_64"]
```

Comment:
Are you sure this is not necessary anymore?

Reply:
these are the only two arches available on this runtime.